ui/app/mainui/AppMain.qml

```diff
@@ -836,8 +836,24 @@ Item {
 
     Settings {
         id: appMainLocalSettings
+        category: "AppMainLocalSettings_%1".arg(allContacsAdaptor.selfPubKey)
```

Comment:
isn't it more readable using modern js syntax like
```
`AppMainLocalSettings_${allContacsAdaptor.selfPubKey}`
```
?

Reply:
Well... matter of taste probably, I find the backtick syntax _very_ confusing and unreadable